.travis.yml

```diff
@@ -1,17 +1,26 @@
 sudo: false
+cache: bundler
 language: ruby
 
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2
-  - jruby
-  - rbx-2
+  - 2.0.0-p648
+  - 2.1.8
+  - 2.2.4
+  - 2.3.0
```

Comment:
>> Can this be merged or close it?

I'll clarify with the rest of the team today and then we'll probably merge it (says my gut feeling ;) )